src/components/chromecast/chromecastplayer.js

```diff
@@ -211,19 +189,16 @@ define(['appSettings', 'userSettings', 'playbackManager', 'connectionManager', '
      * session update listener
      */
     CastPlayer.prototype.sessionUpdateListener = function (isAlive) {
-
-        //console.log('sessionUpdateListener alive: ' + isAlive);
-
         if (isAlive) {
-        }
-        else {
+            console.log('sessionUpdateListener: not alive');
```

Comment:
either variable is inverted logic, or log message is wrong

Reply:
forgot an exclamation mark, you're right